src/_includes/docs/install/compiler/xcode.md

```diff
@@ -98,7 +92,26 @@ To [install and set up CocoaPods][cocoapods], run the following commands:
    {{prompt1}} brew install cocoapods
    ```
 
-Using Homebrew reduces potential issues with chipsets and install permissions.
+1. Add CocoaPods to your macOS `PATH` using the following command.
+
+    ```terminal
+    $ cat <<EOF >> ~/.zshenv
+    # Configure CocoaPods
+
+    export GEM_HOME=\$HOME/.gem
+    if [[ \$PATH != *"\$GEM_HOME"* ]]
+    then
+        export PATH=\$GEM_HOME/bin:\$PATH
+    fi
+    EOF
+    ```
+
+1. Load the changes to your current open terminal sessions
+   using the following command.
+
+    ```terminal
+    $ source ~/.zshenv
+    ```
```

Comment:
I think we should just leave this out, we could just say "Restart your terminal".

Reply:
If you're like me, having to restart 5-6 terminals would be... unpleasant. This command is a shorter way to do this.

Reply:
Why? Don't you just need to close the terminal window and open a new one?

Reply:
I run [Tmux](https://github.com/tmux/tmux/wiki) and have multiple tabs open in my Terminal app.

Reply:
If someone is advanced enough to use Tmux, they should be advanced enough to know how to handle this scenario